app/knight.py

```diff
@@ -0,0 +1,35 @@
+from equiption.armour import Armour
+from equiption.potion import Potion
+from equiption.weapon import Weapon
+
+
+class Knight:
+    def __init__(self,
+                 name: str,
+                 power: int,
+                 hp: int,
+                 armour: list = None,
```

Comment:
The `armour` parameter is expected to be a list of dictionaries, but the current implementation of `Armour` class expects a list of dictionaries with specific keys like 'part' and 'protection'. Ensure that the input format matches the expected structure.